source/reference/mongosync-behavior.txt

```diff
@@ -44,107 +47,9 @@ line options.
    - For a full list of limitations, see :ref:`Cluster-to-Cluster Sync
      Limitations<c2c-limitations>`. 
 
-.. _c2c-cli-options:
-
-Options
--------
-
-.. |opt-term| replace:: option
-
-Global Options
-~~~~~~~~~~~~~~
-
-.. option:: --cluster0 <URI>
-
-   .. include:: /includes/opts/cluster0.rst
-
-   To set the ``--cluster0`` option from a configuration file,
-   see the :setting:`cluster0` setting.
-
-.. option:: --cluster1 <URI>
-
-   .. include:: /includes/opts/cluster1.rst
-
-   To set the ``--cluster1`` option from a configuration file,
-   see the :setting:`cluster1` setting.
-
-.. option:: --config <filename>
-
-   Sets the path to the configuration file.
-
-   For more information, see :ref:`c2c-mongosync-config`.
-
-.. option:: --disableTelemetry
-
-   .. versionadded:: 1.4.0
-
-   .. include:: /includes/opts/disableTelemetry
-
-   To set the ``--disableTelemetry`` option from a configuration file,
-   see the :setting:`disableTelemetry` setting.
-
-   For more information, see :ref:`User Data Collection
-   <c2c-telemetry>`.
-
-.. option:: --help, -h
-
-   Prints usage information to stdout.
-
-.. option:: --id <ID>
-
-   .. include:: /includes/opts/id.rst
-
-   To set the ``--id`` option from a configuration file,
-   see the :setting:`id` setting.
-
-.. option:: --loadLevel <level>
-
-   *Type*: integer
-
-   .. include:: /includes/opts/loadLevel.rst
-
-   To set the ``--loadLevel`` option from a configuration file, see the
-   :setting:`loadLevel` setting.
-
-   .. include:: /includes/opts/loadlevel-warning.rst
-   
-.. option:: --logPath <DIR>
-
-   .. include:: /includes/opts/logPath.rst
-
-   To set the ``--logPath`` option from a configuration file,
-   see the :setting:`logPath` setting.
-
-   .. note:: 
-
-      .. include:: /includes/fact-log-rotation-usr1-signal
-
-.. option:: --port
-
-   .. include:: /includes/opts/port.rst
-
-   To set the ``--port`` option from a configuration file,
-   see the :setting:`port` setting.
-
-.. option:: --verbosity <level>
-
-   .. |verbosity-opt-type| replace:: option
-   .. |verbosity-opt| replace:: ``--verbosity``
-
-   .. include:: /includes/opts/verbosity.rst
-
-   To set the ``--verbosity`` option from a configuration file,
-   see the :setting:`verbosity` setting.
-   
-.. option:: --version, -v
-
-   Prints ``mongosync`` version information to stdout.
-
 Behavior
```

Comment:
[question] Is there a way to group or organize the different behavior sections on the page? Right now, it seems like the order is a bit arbitrary so I wonder if we can group some of these under parent headings (for ex, Reads and Writes as a parent over Read and Write Concern, Read Preference, Write Blocking, etc)?

Reply:
I like this idea. I created three sections: Considerations, Reads and Writes, and Destination Cluster Changes. What are your thoughts?

Reply:
Is there something more specific than "Considerations" that we can use? I would assume that all of the sections on this page are considerations.

Reply:
Alternatively, if there is a "miscellaneous" section of things that don't fit into other specific categories, I think we can put that section at the end instead of the beginning.

Reply:
Ok, I added "Cluster Types" and "Performance." Since "Capped Collections" was the only section left over, I moved it under "Destination Cluster Changes" because it does discuss temporary changes. I am not sure if this was the right call. Thoughts?